doc/man1/openssl-pkcs12.pod

```diff
@@ -76,22 +76,16 @@ by default.
 The filename to write certificates and private keys to, standard output by
 default.  They are all written in PEM format.
 
-=item B<-passin> I<arg>
+=item B<-passin>, I<arg>, B<-passout> I<arg>
 
-The PKCS#12 file (i.e. input file) password source. For more information about
-the format of B<arg>
+The password source for the input and output file.
+For more information about the format of B<arg>
 see L<openssl(1)/Pass phrase options>.
 
-=item B<-passout> I<arg>
-
-Pass phrase source to encrypt any outputted private keys with. For more
-information about the format of B<arg> see the B<PASS PHRASE ARGUMENTS> section
-in L<openssl(1)>.
-
 =item B<-password> I<arg>
 
-With -export, -password is equivalent to -passout.
-Otherwise, -password is equivalent to -passin.
+With C<-export>, C<-password> is equivalent to C<-passout>;
+it is equivalent to C<-passin>.
```

Comment:
Use `B<` rather than `C<` here

Reply:
Also, `s/it is equivalent/otherwise, it is equivalent/`
The sentence looks weird the way it's currently written.

Reply:
fixed.